_health-care/_js/_form.js

```diff
@@ -1284,6 +1333,7 @@ var HealthApp = (function() {
     xmlFieldMap: xmlFieldMap,
     getFormRoot: getFormRoot,
     saveForm: saveForm,
+    validate: validate,
```

Comment:
I had to make this function accessible in order to run the specs on validations, but I wasn't sure if this is the right way to do it. Wanted to check with you @awong-dev.